### ISSUE TITLE
Question about apply_to_masks in DualTransform

### ISSUE THREAD

**Issue by WSYANGSX** [albumentations-team/AlbumentationsX#105]
Hello!
When i was reading the source code about AlbumentationsX, I have a question about apply_to_masks() method in DualTransform.

<img width="1370" height="685" alt="Image" src="https://github.com/user-attachments/assets/a70a00fe-37f5-43c2-a910-8134cd9f2611" />
I find that when masks is empty, apply_to_masks() method will return it directly. But for geometric transform, the shape of img have be changed from (H, W) to (h, w). If we do not change the shape of masks, it will have an impact on the subsequent data splicing for pytorch DataLoader. I don't know if I overlooked the code for shape transformation. 
Thank you for your answer.

**Comment by ternaus** [comment]
@WSYANGSX Thank you, fixed. Also vectorized apply_to_masks for crops and flips. Should work faster now.
